guides/SYNTAX.md

```diff
@@ -77,13 +77,13 @@ let add (x, y: Nat) : IO Nat = do {
 You can also define functions with `let` and `fn`.
 
 ```rust
-let add = Nat -> Nat -> Nat = fn(x, y) => x + y
+let add := Nat -> Nat -> Nat := fn(x, y) => x + y
```

Comment:
I think to keep it consistent with the other type definitions:

The above line should look like this:

```rs
let add : Nat -> Nat -> Nat := fn(x, y) => x + y
```

Reply:
Yes, It's wrong.